testing_app/integration_test/driver.dart

```diff
@@ -4,20 +4,4 @@
 
 import 'package:integration_test/integration_test_driver.dart';
 
-Future<void> main() {
-  return integrationDriver(
-    responseDataCallback: (data) async {
-      // If the tests reported any data, save it to the disk.
-      if (data != null) {
-        for (var entry in data.entries) {
-          print('Writing ${entry.key} to the disk.');
-          // Default storage destination is the 'build' directory.
-          await writeResponseData(
-            entry.value as Map<String, dynamic>,
-            testOutputFilename: entry.key,
-          );
-        }
-      }
-    },
-  );
-}
+Future<void> main() => integrationDriver();
```

Comment:
Since the use of `traceAction` is removed, there are no complete timelines generated, just their summaries. Hence, the custom implementation of `responseDataCallback` isn't required.